packages/cli/src/site-configs.types.ts

```diff
@@ -9,9 +9,9 @@ export type BaseSiteConfig = {
     preloginEnabled?: boolean;
     public?: Record<string, unknown>;
 };
-export type ExtractPrivateSiteConfig<S extends BaseSiteConfig> = S & {
+export type ExtractPrivateSiteConfig<S extends BaseSiteConfig> = Omit<S, "public"> & {
```

Comment:
Why remove it from the private site config? Shouldn't everything be included in it? Like NEXT_PUBLIC_*. variables are also available on the server?

Reply:
Nevermind, my bad. It's still added.